### PR TITLE
[MicroWin] First Anniversary Special PR

### LINKED ISSUE #2697: MicroWin changes explorer look.
Hello I created an iso of windows pro 24h2 with microwin and I notused that it changed the look of explorer . How do i revert it back to the default windows 11 explorer look?

### LINKED ISSUE #2705: MicroWin - Add mstsc (rdp-client)
**Is your feature request related to a problem? Please describe.**
RDP works as a server but not as the client.

**Describe the solution you'd like**
keep all mstsc-functionality on microwin

**Describe alternatives you've considered**
copying mstsc.exe to system32 works.. but it lacks some features. please just keep this part of windows in microwin.

### LINKED ISSUE #2848: 24H2 Microwin and AMD chipset drivers
Hello, I cant install AMD chipset drivers when 24H2 ISO is done with microwin. Error message pops up "This program is meant for an AMD system. The program will be terminated because of system requirements not met"
If I take original ISO, install works OK.

## To Reproduce
0. Must have AMD chipset
1. make 24H2 ISO with microwin 
2. install AMD chipset drivers from amd.com

## Expected behavior
Error  "This program is meant for an AMD system. The program will be terminated because of system requirements not met"

### LINKED ISSUE #2885: File Explorer Issues on 24H2
## Describe the bug
Windows File Explorer misbehaving on a fresh install of MicroWin Windows 11 Pro 24H2 image
-Unable to use tabs feature
-Clicking on breadcrumbs creates new instance instead of changing directory (conditional, might be hard to reproduce)

## To Reproduce
1. Install MicroWin W11P 24H2 image
2. Open File Explorer
3. Try and open a new tab
4. Try clicking on breadcrumbs

## Additional context
Possibly might be related to certain tweaks (Setting services to manual perhaps)
Can't get the breadcrumbs issue to work reliably

### PR DESCRIPTION
**Pull request by CodingWonders**
## Type of Change
- [x] New feature
- [x] Bug fix
- [x] UI/UX improvement

## Description
This PR adds final fixes and enhancements for this first year of MicroWin, and is, therefore, an anniversary special. MicroWin was first introduced to the WinUtil repository as a PR by @KonTy on October 26, 2023 (the 11th anniversary of Windows 8!), on #1129, and has been improved ever since.

The current highlights are these:

- **You can now download 24H2 from within WinUtil**. Simply select "24H2" and let the ISO downloader download it for you:

<p align="center">
  
</p>

> [!NOTE]
> Downloads for older versions of Windows 11 have been removed from WinUtil because Fido no longer supports downloading them. Microsoft has removed the links for these (obviously in favor of 24H2)

- **Remote Desktop Connection is no longer removed on 24H2 and newer**

<p align="center">
  
</p>

- **You can now perform operations to capabilities (FoDs)**. Before this change, capability metadata would be removed on 24H2 and Selenium builds of Windows as a Service (WaaS). That no longer happens and you can now perform operations with capabilities:

<p align="center">
  
</p>

- **MicroWin gives (*slightly*) better results on Windows 10**
- **Error reporting for package removal has been significantly improved**, with the suggestion from and help of @og-mrk (https://github.com/CodingWonders/winutil/pull/1 and https://github.com/CodingWonders/winutil/pull/2):

<p align="center">
  
</p>

- **It no longer messes up the look of the File Explorer in 24H2 and later:**

<p align="center">
  
</p>

- **The volume mixer now retains the application volume**, thanks to @MyDrift-user (originally on #2856):

<p align="center">
  
</p>

## Testing
Testing has concluded successfully thus far.

## Impact
A better experience

## Issue related to PR

- Resolves #2697
- Resolves #2705
- Resolves #2848
- Resolves #2885

**NOTE:** to be filled

## Additional Information
None

## Checklist
- [X] My code adheres to the coding and style guidelines of the project.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

**Comment by CodingWonders** [comment]
PowerShell 7 support is temporarily broken with the error "Specified method is not supported". Look at the additional notes for more information

**Review by ChrisTitusTech — APPROVED** [review]

File: functions/private/Invoke-WinUtilMicroWin-Helper.ps1 [hunk 1]
```diff
@@ -98,6 +109,8 @@ function Remove-Features() {
             $_.FeatureName -NotLike "*Media*" -AND
             $_.FeatureName -NotLike "*NFS*" -AND
             $_.FeatureName -NotLike "*SearchEngine*" -AND
+            $_.FeatureName -NotLike "*RemoteDesktop*" -AND
+            $_.FeatureName -NotLike "*Recall*" -AND
```
Comment (ChrisTitusTech) [inline functions/private/Invoke-WinUtilMicroWin-Helper.ps1:113]: Having to add recall back in because of file explorer reversions is a bit strange.

**Comment by Lymsleya** [comment]
I came back.. For me the sound problem still persists, it drives me insane it's so annoying to deal with to the point where i wouldn't use winutil but i want to i love it. No other problems just this.

**Comment by YggdrasziI** [comment]
> I came back.. For me the sound problem still persists, it drives me insane it's so annoying to deal with to the point where i wouldn't use winutil but i want to i love it. No other problems just this.

Still the case

**Comment by YggdrasziI** [comment]
Thanks for your time i will look at it !

**Pull request merged**